server/market/routers_test.go

```diff
@@ -106,14 +106,29 @@ func (a *TAuth) getSend() *msgjson.Message {
 
 type TMarketTunnel struct {
 	adds       []*orderRecord
+	auth       *TAuth
 	midGap     uint64
 	locked     bool
 	watched    bool
 	cancelable bool
 }
 
 func (m *TMarketTunnel) SubmitOrder(o *orderRecord) error {
+	// set the server time
+	now := time.Now().Unix()
+	o.order.SetTime(now)
+
 	m.adds = append(m.adds, o)
+
+	// Send the order, but skip the signture
+	oid := o.order.ID()
+	resp, _ := msgjson.NewResponse(1, &msgjson.OrderResult{
+		Sig:        msgjson.Bytes{},
+		ServerTime: uint64(now),
+		OrderID:    oid[:],
+	}, nil)
+	m.auth.Send(account.AccountID{}, resp)
+
```

Comment:
BTW, I'm going to revisit the order router tests after testifying any new stuff in this PR.  I suspect the PR changes and these test hacks have made some of the routers_test.go less useful.  But I wanted to get the tests working and gain a little better familiarity with what you have here.